src/java/org/apache/cassandra/index/sai/disk/v1/SSTableIndexWriter.java

```diff
@@ -261,9 +261,8 @@ private void flushSegment() throws IOException
         try
         {
             long bytesAllocated = currentBuilder.totalBytesAllocated();
-
+            logger.debug("Flushing after {} allocated", FBUtilities.prettyPrintMemory(bytesAllocated));
```

Comment:
is it possible to add a reference to the file ? (in CNDB we configure the logger at "debug" level, so this line will be printed)

Reply:
I'm going to move this to a different PR, but the file is logged at the start of the index build and I don't think it's useful to include it redundantly in every subsequent message.